Person.java

```diff
@@ -0,0 +1,48 @@
+public class Person {
+    String name="unknown";
+    int age;
+    int id;
+    boolean parent_permission=true;
+
+    public Person(String  name,int id,int age,boolean parent_permission){
```

Comment:
- You did a great effort but you did not set default value for name and parent_permission